packages/interface/src/layouts/AppContainer.tsx

```diff
@@ -0,0 +1,23 @@
+import { type PropsWithChildren } from "react";
+
+import { cn } from "~/utils/classNames";
+
+type IAppContainerProps = PropsWithChildren<{ as?: React.ElementType; className?: string; fullWidth?: boolean }>;
+
+export const AppContainer = ({
+  children = null,
+  as: Component = "div",
+  className = "",
+  fullWidth = false,
+  ...props
+}: IAppContainerProps): JSX.Element => (
+  <Component
+    className={cn("mx-auto flex w-full", className, {
+      "max-w-screen-2xl": !fullWidth,
```

Comment:
maybe this `mx-auto` causing the Footer having no `margin-x` if the screen size is not large (e.g. my macbook 14)